CHANGES.rst

```diff
@@ -10,8 +10,18 @@ New features
 - conda installable package
 - GPU version
 
+
 2.4.0 (upcoming)
-==================
+================
+This release adds the boxsize parameter to the command line interfaces and
+requires the user to specify boxsize in the Python interfaces to the periodic
+theory functions.  It also contains a number of performance and quality-of-life
+improvements.
```

Comment:
Can you please re-phrase to "It also contains a number of improvements for performance and code quality"